package.json

```diff
@@ -111,11 +116,11 @@
     "js:allBreaches": "minify public/js/all-breaches/all-breaches.js -o public/dist/all-breaches.min.js",
     "css:minify": "cleancss -o public/dist/app.min.css public/dist/app.min.css",
     "css:concat": "concat -o public/dist/app.min.css public/css/*.css",
-    "build:css": "run-s css:concat css:minify",
+    "build:css": "./node_modules/.bin/gulp build",
```

Comment:
npm scripts should sneakily put `./node_modules/.bin/` in the path for you, so you can probably just use `gulp build` here.

Reply:
Revised!